examples/basic/basic.c

```diff
@@ -13,6 +13,8 @@
 #include <stdio.h>
 #include <string.h>
 
+#include "utils_common.h"
```

Comment:
should we really use utils in examples? it will be harder to make them standalone

Reply:
I can define `util_strncpy()`  inside `basic.c`